songs/joli-rouge.txt

```diff
@@ -1,5 +1,6 @@
 X: 1
 T: Joli Rouge
+%%vocalfont times 10
```

Comment:
I might just reach into `abc.js` and make the default this font, cause the default is hideous.

Reply:
It really is.
Perhaps we have a space for "common ABC props"?

Can preload with this kind of stuff:
```
[I: MIDI=program 20]
%%titlefont harrington 40
%%vocalfont harrington 10
%%gchordfont harrington 12
%%notecolors 1
%%notelabels 1
```

Actually, I think that's a good idea. I'll open an issue.